src/Inkluzitron/Modules/UserModule.cs

```diff
@@ -102,16 +104,43 @@ public async Task ShowUserInfoAsync([Name("uživatel")] IUser user = null)
 
             var guildUser = await UsersService.GetUserFromHomeGuild(user);
             if (guildUser != null)
-                descBuilder.Append("**Na serveru od:** ").AppendLine(guildUser.JoinedAt.Value.LocalDateTime.ToShortDateString());
+                descBuilder.AppendLine($"**V Bobánkově od:** {guildUser.JoinedAt.Value.LocalDateTime:d. M. yyyy}");
             else
-                descBuilder.AppendLine("Již není na serveru");
+                descBuilder.AppendLine("Již není v Bobánkově");
+
+            var inviteUsed = await DbContext.Invites.AsQueryable()
+                .Where(i => i.UsedByUserId == user.Id)
```

Comment:
Na tom sloupci tuším není index, snad to nás databázový server zvladne

Reply:
Na tom sloupci není index, protože jeden uživatel může "použít" víc invites (př. když bude odejit a potom ho někdo znovu pozve)